CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.8.16] - 2023-05-18
+
+### 🚀 New Features and Enhancements
+
+- Add setup warning about latest tested version  [#3895](https://github.com/iterative/vscode-dvc/pull/3895) by [@julieg18](https://github.com/julieg18)
+- Add upgrade dvc button to setup when incompatible [#3904](https://github.com/iterative/vscode-dvc/pull/3904) by [@julieg18](https://github.com/julieg18)
+
```

Comment:
Hmmm, I figured adding a label to #3931 would do the trick but it still doesn't appear in the change log. Technically, not a huge deal, if the log is missing a pr but I'll go ahead and update the change log manually.